package-lock.json

```diff
@@ -1,7 +1,11595 @@
 {
   "name": "cactbot",
+  "lockfileVersion": 2,
```

Comment:
But when I read the doc of npm, it said version 2 should be backwards compatible with version 1.

https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json#lockfileversion

Which version of npm are you using, maybe you need an upgrade?

Reply:
It is ... sort of backwards compatible? npm says "I'll try my best" and prints out a warning.  It seems to work, but that also seems fragile.  My npm is 6.14.8.

Regardless, I think if you want to upgrade the lockfile, it'd be better to do that in a separate PR where nothing else changes but updating the lockfile version.  It makes the diff impossible to read if you are both bumping the version and adding new packages.

Reply:
Well, I just don't know how to prevent `npm` to commit the newer lockfile version, and I have not found solutions on Internet (maybe my searching keyword is wrong?)
Maybe I should manually revert this.

And sorry that I'm busy these days, didn't respond so long.

Reply:
```diff
diff --git a/package-lock.json b/package-lock.json
index 5af70bb36..d0abd338c 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -492,9 +492,9 @@
       "dev": true
     },
     "@types/node": {
-      "version": "13.1.1",
-      "resolved": "https://registry.npmjs.org/@types/node/-/node-13.1.1.tgz",
-      "integrity": "sha512-hx6zWtudh3Arsbl3cXay+JnkvVgCKzCWKv42C9J01N2T2np4h8w5X8u6T
pz5mj38kE3M9FM0Pazx8vKFFMnjLQ==",
+      "version": "14.14.43",
+      "resolved": "https://registry.npmjs.org/@types/node/-/node-14.14.43.tgz",
+      "integrity": "sha512-3pwDJjp1PWacPTpH0LcfhgjvurQvrZFBrC6xxjaUEZ7ifUtT32jtjPxEM
Mblpqd2Mvx+k8haqQJLQxolyGN/cQ==",
       "dev": true
     },
     "@types/normalize-package-data": {
```

Is the diff in lockfile version 1, if that helps? Maybe a separate PR to talk about bumping to lockfile version 2 and what that means.